howdoi/errors.py

```diff
@@ -1,10 +1,14 @@
+import logging
+
+
 class GoogleValidationError(Exception):
-    pass
+
+    logging.exception('Google Validation Error %s ', Exception)
```

Comment:
I believe this code will run when the file is compiled? I see a lot of this in the logs:

```
ERROR:root:Bing Validation Error <class 'Exception'>
```

Instead, we will only want to print this out when the class is used.

Reply:
Call this in the`__init__`

Reply:
Changing this to use constructors